libs/sketch/.storybook/main.ts

```diff
@@ -1,7 +1,7 @@
 import type { StorybookConfig } from '@storybook/angular';
 
 const config: StorybookConfig = {
-  stories: ['../**/*.stories.@(js|jsx|ts|tsx|mdx)'],
+  stories: ['../**/*.@(mdx|stories.@(js|jsx|ts|tsx))'],
```

Comment:
is there a reason you moved it or was it done automatically?

Reply:
was done by the storybook migration script. Something with the markdown files changed after update to v8